examples/fleet/kuard/README.md

```diff
@@ -0,0 +1,184 @@
+# Example: Exporting service using Fleet resource placement
+
+This directory contains the instructions on how to export services using Fleet resource placement and setup Layer 4 load balancing across workloads deployed across fleet member clusters.
+
+The application is using [kuard](https://github.com/kubernetes-up-and-running/kuard) as the service to demonstrate the responding cluster/pod.
+
+## Before you begin
+* Install [kubectl](https://kubernetes.io/docs/tasks/tools/).
+* Create a fleet with three members: `aks-member-1`, `aks-member-2`, and `aks-member-3` using [Azure CNI networking](https://review.learn.microsoft.com/en-us/azure/aks/configure-azure-cni).
+* Member clusters reside on the same virtual network, or [peered virtual networks](https://docs.microsoft.com/en-us/azure/virtual-network/virtual-network-peering-overview) such that pods from different clusters can communicate directly with each other using pod IPs.
+* Configure kubectl to communicate with your hub cluster.
+* Get `kubeconfig` files for member clusters as `member1`, `member2`, and `member3`.
+
+## Objectives
+* Deploy a Kuard demo application with an internal load balancer service and export the service in the hub cluster.
+* Place the resources (deployment, service, service export) to member clusters.
+* Expose fleet-wide endpoints from exported services with a multi-cluster service.
+
+## Steps
+
+### 1. Deploy to the Hub Cluster
+
+1. Switch to the hub cluster context to create namespace:
+
+   ```shell
+   kubectl create namespace kuard-demo
+   ```
+
+2. Deploy a Kuard demo application with an internal load balancer service and export the service.

```

Comment:
does it have to have this annotation

service.beta.kubernetes.io/azure-load-balancer-internal: "true"

Reply:
the typical workflow is to expose the internal service and the service is behind the internal load balancer. Thus, the annotation is required and examples/fleet/kuard/kuard-export-service.yaml has already had this annotation.